Dockerfile

```diff
@@ -1,10 +1,17 @@
-FROM gidden/messageix-base
+FROM openjdk:11-slim-buster
 
-COPY . /ixmp
-WORKDIR /
-ENV IXMP_PATH /ixmp
-ENV IXMP_R_PATH /ixmp/ixmp
-RUN pip2 install xlsxwriter xlrd
-RUN pip3 install xlsxwriter xlrd
-RUN cd /ixmp && python2 setup.py install 
-RUN cd /ixmp && python3 setup.py install
+# installing python 3
+ARG VIRTUAL_ENV=/opt/python3
+RUN apt-get update && apt-get install -y --no-install-recommends curl unzip graphviz python3 python3-pip && \
+    pip3 install virtualenv && \
+    virtualenv -p python3 $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+
+# installing GAMS
+ARG GAMS_VERSION=27.3.0
```

Comment:
We need to make it clear, somewhere, that when the GAMS version is incremented in the other CI scripts, it should also be done here, or vice versa. ci/env.sh currently uses GAMS 25.1.1.

At a minimum, comments in both places like `# See also ci/env.sh` and `# See also Dockerfile`.